docs/proposals/201812_thanos-remote-receive.md

```diff
@@ -0,0 +1,166 @@
+## Thanos Remote Write
+
+Status: draft | *in-review* | rejected | accepted | complete
+
+Implementation Owner: @brancz
+
+## Summary
+
+This document describes the motivation and design of the Thanos receiver component, as well as how it fits into the rest of the Thanos ecosystem and components.
+
+## Motivation
```

Comment:
Might be worth outlining the scenarios where we do not see this as an appropriate solution e.g. the standard use case of Thanos where sidecar should suffice. Mainly single tenant application where team owns whole stack and can therefore allow inbound traffic to sidecar.

Reply:
Good idea! Will do.

Reply:
I would propose to mention/link something like: https://docs.google.com/document/d/1H47v7WfyKkSLMrR8_iku6u9VB73WrVzBHb2SB6dL9_g/edit# (: It kinds of relates to risks that comes with push model. 

I think it's important to say that you should use this component to transfer *data from other Prometheuses*, not to push arbitrary metrics. That's the most important  abuse risk I think.